Common/src/main/java/edu/sdccd/cisc191/template/VegetableFacts.java

```diff
@@ -9,5 +9,5 @@ public interface VegetableFacts {
      * Returns information about a vegetable
      * @return information about a vegetable
      */
-    public String VegetableFacts();
+    String VegetableFacts();
```

Comment:
The name for this method should probably be changed to something other than the name of the class that it's in. Maybe just make the first letter of the method name lower case. Other usages of this method throughout the program should have the name also changed to the lower case letter.